tests/boot/with_mcumgr/sysbuild.cmake

```diff
@@ -0,0 +1,16 @@
+# Add the mcuboot key file to the secondary swapped app
+# This must be done here to ensure that the same key file is used for signing
+# both the primary and secondary apps
+set(smp_svr_CONFIG_MCUBOOT_SIGNATURE_KEY_FILE
+    \"${SB_CONFIG_BOOT_SIGNATURE_KEY_FILE}\" CACHE STRING
+    "Signature key file for signing" FORCE)
```

Comment:
I don't see why this is needed, this should be done automatically by the sysbuild code if MCUboot is enabled

Reply:
I was copied it from test `tests/boot/test_mcuboot`
https://github.com/zephyrproject-rtos/zephyr/blob/3e2765cc0da9f309b952ee31c9ff7cf6ad0a3b7a/tests/boot/test_mcuboot/sysbuild.cmake#L7

This is a secondary swapped app and when I removed this key file assignment, then I received:
`smp_svr/zephyr/.config:CONFIG_MCUBOOT_SIGNATURE_KEY_FILE=""`,
so it is not done automatically.